### DISCUSSION TITLE
🎨 Agregar leyendas de color de línea a Línea de tiempo

### DISCUSSION THREAD

**Discussion by anattolia** [enflujo/imagina-ninezya#85] [Ideas]
Agregar leyendas y cambiar colores de líneas a Línea de tiempo para saber cuál es cuál.

**Comment by 1cgonza** [comment]
@Juliansintilde Esto cómo va? alguna propuesta? es urgente definir el sistema de leyendas de la línea de tiempo.

**Comment by Juliansintilde** [comment]
Hay una propuesta relacionada con esto en el #78 
Así las leyendas se reemplazan por el color del marco al elegir el departamento. 
El color del borde del marco, de la letra en el nombre y del borde del municipio o departamento en el mapa, serían el mismo. Creo que con eso es claro cuál es cuál en la línea de tiempo.

La línea nacional es con ` --fucsiaEsmalte: #af067d;`
La lista de los colores de las cajas de departamentos **ya está cargada en estilos** y es esta:
```
 // Colores línea de tiempo
  --color1: #1818ff;
  --color2: #ff771a;
  --color3: #ff00ff;
  --color4: #10b5ae;
  --color5: #b4b800;
  --color6: #570a8f;
  --color7: #e40074;
  --color8: #762e00;
  --color9: #ffceff;
  --color10: #03ffff;
  --color11: #ff0400;
  --color12: #557eff;
  --color13: #ffc720;
  --color14: #ad9987;
  --color15: #ffa8b1;
  --color16: #ea6eff;
  --color17: #007876;
  --color18: #dffd00;
  --color19: #7dffa2;
  --color20: #7d57a2;
  --color21: #ffb46d;
  --color22: #1b1a1a;
  --color23: #af6c00;
  --color24: #4dcc23;
  --color25: #24467d;
  --color26: #e0ff21;
  --color27: #8ca2ff;
  --color28: #d35446;
  --color29: #cdeab6;
  --color30: #bf3dc5;
  --color31: #59708c;
  --color32: #cfb428;
```

![5_lineadetiempo_seleccion](https://github.com/enflujo/imagina-ninezya/assets/42554838/efbb1cef-6e78-48da-b46b-5a8410d71308)
